services/static-webserver/Dockerfile

```
@@ -0,0 +1,30 @@
+FROM joseluisq/static-web-server:1.15.0-alpine as base
+
+LABEL maintainer=neagu@itis.swiss
+
+# simcore-user uid=8004(scu) gid=8004(scu) groups=8004(scu)
+ENV SC_USER_ID=8004 \
+    SC_USER_NAME=scu \
+    SC_BUILD_TARGET=base \
+    SC_BOOT_MODE=default
+
+RUN adduser \
+    --uid ${SC_USER_ID} \
+    --disabled-password \
+    --gecos "" \
+    --shell /bin/sh \
+    --home /home/${SC_USER_NAME} \
+    ${SC_USER_NAME}
```

```suggestion
adduser -D -u ${SC_USER_ID} -s /bin/sh -h /home/${SC_USER_NAME} ${SC_USER_NAME}
```
I'm not sure what you have currently is alpine compatible. please check